src/utils/navigation.ts

```diff
@@ -129,14 +130,19 @@ export const NAVIGATION: Navigation = {
     previousPage: () => config.INTERRUPT_CARD_URL,
     nextPage: [config.PRESENTER_URL]
   },
+  [config.OVERSEAS_NAME_WITH_PARAMS_URL]: {
+    currentPage: config.OVERSEAS_NAME_PAGE,
+    previousPage: () => config.INTERRUPT_CARD_URL,
+    nextPage: [config.PRESENTER_URL]
+  },
   [config.PRESENTER_URL]: {
     currentPage: config.PRESENTER_PAGE,
     previousPage: () => config.OVERSEAS_NAME_URL,
     nextPage: [config.WHO_IS_MAKING_FILING_URL]
   },
   [config.PRESENTER_WITH_PARAMS_URL]: {
     currentPage: config.PRESENTER_PAGE,
-    previousPage: () => config.OVERSEAS_NAME_URL,
+    previousPage: (appData: ApplicationData, req: Request) => getUrlWithParamsToPath(config.OVERSEAS_NAME_WITH_PARAMS_URL, req),
```

Comment:
This is why 'previousPage' function needed the request object - to build the correct link when back link is being rendered following a validation failure.